### ISSUE TITLE
Background video on home page does not have a caption track

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/murrow.wsu.edu#125]

**Comment by jeremyfelt** [comment]
There is no audio to transcribe, but I think we need at least one track with a description of what's going on in the video.
